.github/workflows/get_license.yml

```diff
@@ -0,0 +1,21 @@
+name: Acquire activation file
+on:
+  workflow_dispatch:
+
+jobs:
+  activation:
+    name: Request manual activation file 🔑
+    runs-on: ubuntu-latest
+    steps:
+      # Request manual activation file
+      - name: Request manual activation file
+        id: getManualLicenseFile
+        uses: game-ci/unity-request-activation-file@v2.0-alpha-1
```

Comment:
I notice game.ci's example uses just v2 (amongst other small differences), does it matter?

Reply:
No :-) 

https://github.com/game-ci/unity-request-activation-file/compare/v2.0-alpha-1...v2 

But it'd be cleaner with just v2, even though they didn't do a release for that, just a tag.